schedule/yam/agama_mediacheck.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
we don't need to use previous nomenclature, `agama_check_installation_medium`. Let's apply it everwhere.